frontend/chota-imambara-explorer/style.css

```
@@ -0,0 +1,889 @@
+  --chota-sand: #e8e4d9;
+  --chota-glass: rgba(26, 35, 50, 0.55);
+  --chota-glass-strong: rgba(15, 20, 25, 0.82);
+  --chota-border: rgba(212, 168, 75, 0.2);
+  --chota-navbar-height: 70px;
+}
+
+/* Page Main Container */
+.chota-main {
+  background-color: var(--chota-night);
+  color: #f1f5f9;
+  font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)